[no-issue] Announcment clean up app enablement (#2913)
Cumulocity-IoT/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md

ticket: MTM-56466
version: 1020.0.0
---

> The redesigned dashboards with both visual and functional enhancements as well as new functionalities has been implemented:

Reviewer: ```suggestion
Redesigned dashboards with both visual and functional enhancements as well as new functionalities have been implemented:
```
